### PR TITLE
Publish correct transforms from SLAM.

### PR DESCRIPTION
**Pull request by wohe**

### CODE REVIEW

File: cartographer_ros/src/cartographer_node_main.cc [hunk 1]
```diff
@@ -524,26 +524,26 @@ void Node::PublishSubmapList(const int64 timestamp) {
 void Node::PublishPose(const int64 timestamp) {
   const ::cartographer::common::Time time =
       ::cartographer::common::FromUniversal(timestamp);
-  const Rigid3d tracking_to_map = trajectory_builder_->pose_estimate().pose;
+  const Rigid3d tracking_to_local = trajectory_builder_->pose_estimate().pose;
+  const ::cartographer::mapping::Submaps* submaps =
+      trajectory_builder_->submaps();
+  const Rigid3d local_to_map =
```
Comment (damonkohler) [inline cartographer_ros/src/cartographer_node_main.cc]: local_to_global

File: cartographer_ros/src/cartographer_node_main.cc [hunk 2]
```diff
@@ -524,26 +524,26 @@ void Node::PublishSubmapList(const int64 timestamp) {
 void Node::PublishPose(const int64 timestamp) {
   const ::cartographer::common::Time time =
       ::cartographer::common::FromUniversal(timestamp);
-  const Rigid3d tracking_to_map = trajectory_builder_->pose_estimate().pose;
+  const Rigid3d tracking_to_local = trajectory_builder_->pose_estimate().pose;
```
Comment (damonkohler) [inline cartographer_ros/src/cartographer_node_main.cc]: s/local/local_map

**Comment by damonkohler** [comment]
LGTM

File: cartographer_ros/src/cartographer_node_main.cc (hunk 2 above)
Comment (wohe) [inline cartographer_ros/src/cartographer_node_main.cc]: As discussed offline, we'll instead rename GetOdometryToMapTransform() to GetLocalToGlobalTransform() in future PRs.

**Pull request merged**